kuma/javascript/src/index.jsx

```diff
@@ -17,7 +17,7 @@ if (container) {
     window._react_data = null; // eslint-disable-line camelcase
 
     // Store the string catalog so that l10n.gettext() can do translations
-    localize(data.requestData.locale, data.localizationData);
+    localize(data.requestData.locale, data.stringCatalog, data.pluralFunction);
```

Comment:
In the web browser, we're always going to get the plural function we need passed to us as part of the window._react_data object, so we don't have to construct it ourselves. That's how we avoid the CSP issue.